frontend/src/old-pages/Configure/Create.tsx

```diff
@@ -35,7 +35,7 @@ function handleWarnings(resp: any) {
   if(!resp.validatonMessages)
     return;
 
-  resp.validatonMessages.forEach((message: any, i: any) => {
+  resp.validatonMessages.forEach((message: any) => {
```

Comment:
typo: `validatonMessages` (also above) - is this code working? I don't see any other references to this variable in the codebase

Reply:
It looks like we weren't showing the user the creation warnings previously due to the check in the lines above.

With the fix to the typo I see the warnings from the create call displayed via the notification mechanism:

![image](https://user-images.githubusercontent.com/6087509/182417498-2e906889-49c2-44a4-987e-c8c207e83547.png)